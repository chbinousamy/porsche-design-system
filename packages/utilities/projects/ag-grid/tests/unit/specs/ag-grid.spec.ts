import fs from 'fs';
import path from 'path';
test('should match snapshot in theme light', () => {
  const css = fs.readFileSync(path.resolve(__dirname, '../../../dist/theme-light.css'), 'utf8');
  expect(css).toMatchSnapshot();
});

test('should match snapshot in theme dark', () => {
  const css = fs.readFileSync(path.resolve(__dirname, '../../../dist/theme-dark.css'), 'utf8');
  expect(css).toMatchSnapshot();
});
