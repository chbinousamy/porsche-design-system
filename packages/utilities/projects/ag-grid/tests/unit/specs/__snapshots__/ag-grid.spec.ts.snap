// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should match snapshot in theme light 1`] = `
":where(.ag-root-wrapper, .ag-measurement-container, .ag-apply-theme-variables, .ag-popup) {
  --ag-background-color: #fff;
  --ag-foreground-color: #010205;
}
"
`;
